callback.go

```diff
@@ -46,7 +46,7 @@ func (c *Callback) MessageSig() (string, int64) {
 	if c.IsInline() {
 		return c.MessageID, 0
 	}
-	return c.Message.MessageSig()
+	return c.Message.Message.MessageSig()
```

Comment:
Implement the `Editable` interface for the `InaccessibleMessage`.

```suggestion
	return c.Message.MessageSig()
```